src/spoke/BalanceSheet.sol

```diff
@@ -90,6 +90,7 @@ contract BalanceSheet is Auth, Recoverable, IBalanceSheet, IBalanceSheetGatewayH
         } else {
             IERC6909(asset).transferFrom(msg.sender, escrow_, tokenId, amount);
         }
+        emit Deposit(poolId, scId, asset, tokenId, amount);
```

Comment:
Each time we call a deposit, we will emit `NoteDeposit` and `Deposit`. If we're fine with that, I'm also fine